src/index.ts

```diff
@@ -0,0 +1,12 @@
+import { autoChangelogCommand } from './auto-changelog-config';
+
+// Shared variables used by various configurations.
+const LATEST_VALID_TAG_COMMAND = 'git describe --tags --abbrev=0 --exclude "*rc*" $(git rev-list --tags --max-count=1)';
```

Comment:
Since `--match` and `--exclude` accept a glob (7) pattern (which from what I can tell does not have a way to match "ends with" style patterns) from my testing `--match` will not allow us to get what our requirements infer is a "valid" tag (vX.X.X). So it feels like we are left with excludes.

However, if I change the exclude to a glob matching `v*.*.*-*`, that filters out any type of pre-release suffix. Would this acceptable (assuming we make it configurable of course)?

Reply:
Yeah, that sounds good.